packages/circuit-ui/components/NotificationModal/NotificationModal.tsx

```diff
@@ -179,13 +179,8 @@ export const NotificationModal: ModalComponent<NotificationModalProps> = ({
 
         function wrapOnClick(onClick?: ButtonProps['onClick']) {
           return (event: ClickEvent) => {
-            // FIXME switch to an optional call when apps are on Node 14
-            if (handleClose) {
-              handleClose(event);
-            }
-            if (onClick) {
-              onClick(event);
-            }
+            handleClose?.(event);
```

Comment:
This will need to be transpiled to work with many of our target browsers (older versions of Safari, Chrome, Firefox, Edge). Should we document this (i.e. ask teams to make sure they transpile it) or keep using the old syntax?

Reply:
My thought: we might want to document what target ES version apps should transpile Circuit to in order to work with the intended browser support.

Syntax I can think of that we're using (or probably will be soon):
- optional chaining
- arrow functions
- async/wait
- more?

...but it would be great if we could use these in the repo itself without worrying about browser support

Reply:
I like the idea of suggesting the compile target 👍 We should test whether this means that Circuit UI still requires https://github.com/martpie/next-transpile-modules when used in Next.js.

Reply:
For this specific case, I see that TS actually transpiles this in our common JS build. Here we get:

```js
function wrapOnClick(onClick) {
  return function (event) {
    handleClose === null || handleClose === void 0 ? void 0 : handleClose(event);
    onClick === null || onClick === void 0 ? void 0 : onClick(event);
  };
}
```

@connor-baer do you know how apps pick one of the versions? If they use esm then the optional chaining should be transpiled for the browser support we're looking at

Reply:
I think that depends on the bundler, but I'm not sure.